OpenRA.Mods.TS/Effects/AnimatedBeaconSprite.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using OpenRA.Effects;
+using OpenRA.Graphics;
+using OpenRA.Scripting;
+
+namespace OpenRA.Mods.TS.Effects
+{
+	public class AnimatedBeacon : IEffect
```

Comment:
The initial plan was to use the existing beacon code, but it is so hard-coded to our CTF + remapped rings workaround that I created a new simplified one. I kept the un-hardcoding for documentation and lint testing.